app/representation.py

```diff
@@ -1,26 +1,41 @@
-import random
-from dateutil import parser
 import datetime
+import hashlib
+import random
+
 from app.util import log
+from collections import OrderedDict
+from dateutil import parser
 from events import isSingleDayEvent
-import hashlib
 
-def venueRecord(biz, **details):
+def baseRecord(biz):
     # biz is the response object from the Yelp Search API
 
-    from collections import OrderedDict
-    # h is derived from the providers, but for the main body of the record.
-    # h is for header.
-    h = {
-      "url"        : None,
-      "description": [],
-      "categories" : OrderedDict(),
-      "images"     : [],
-      "hours"      : [],
+    coord = {
+      "lat": biz.location.coordinate.latitude,
+      "lng": biz.location.coordinate.longitude
+    }
+
+    return {
+      "providers": {
```

Comment:
Yeah, this was done as a perf optimization as I noticed it takes _significantly_ longer to update records individually than it does to do a single batch write. We can change it later as you said, but I'll add a comment in the code so this doesn't get overlooked.